### ISSUE TITLE
Widgets within BeginDisabled(false) ... EndDisabled() are unresponsive

### ISSUE THREAD

**Issue by AidanSun05** [ocornut/imgui#4453]
**Version/Branch of Dear ImGui:**

Version: 1.84
Branch: master+docking

**Back-end/Renderer/Compiler/OS**
(irrelevant)

**My Issue/Question:**

When using today's latest commit on both the master and docking branches, anything in between `ImGui::BeginDisabled(false)` and `ImGui::EndDisabled()` is unresponsive to mouse interactions while retaining the default opacity.

**Screenshots/Video**

In the GIF below, you can see that the widgets appear to be enabled, but are unresponsive when I click on them:

![image](https://user-images.githubusercontent.com/61399657/130294048-85dc6703-a5b2-4ce7-b077-4d6ad8d83cd0.gif)

**Standalone, minimal, complete and verifiable example:**
```cpp
// Window:
ImGui::SetNextWindowSize({ 200, 100 });
ImGui::Begin("Disabled Bug");

// This causes the widgets to become unresponsive:
ImGui::BeginDisabled(false);

// Some widgets:
ImGui::Button("Button");

static bool b = false;
ImGui::Checkbox("Checkbox", &b);

ImGui::EndDisabled();
ImGui::End();
```

**Comment by ocornut** [comment]
Fixed with 2e01952b359050d6e4bb3294bfb8eb21b3c0eed7 and b0361375eeb25c37c548abcbca1f500edea6fe90
Issues a new release https://github.com/ocornut/imgui/releases/tag/v1.84.1

**Comment by AidanSun05** [comment]
Thank you for the follow-up! This fixes the bug.
